tests/settings.py

```diff
@@ -46,19 +48,26 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 )
 
-ROOT_URLCONF = 'locking.tests.urls'
+ROOT_URLCONF = 'tests.urls'
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.abspath(os.path.join(BASE_DIR, os.path.pardir, 'assets', 'static'))
 
-TEMPLATE_LOADERS = (
-    'django.template.loaders.filesystem.Loader',
-    'django.template.loaders.app_directories.Loader',
-)
+if django.VERSION < (1, 8):
```

Comment:
Versions less than 1.8 are [no longer supported](https://www.djangoproject.com/download/#supported-versions), so I think we can just delete this block of code instead of wrapping it in a conditional. (I guess we'd need to also remove Django 1.7 from the tox tests as well).